plaidml/bridge/openvino/ops/prior_box.cpp

```diff
@@ -0,0 +1,447 @@
+                        const ngraph::op::PriorBoxAttrs& attrs) {
+  // Center
+  auto C_mix_box = op::concatenate({C_mix, C_mix}, -1);
+  auto C = op::repeat(op::unsqueeze(C_mix_box, {-2})).count(conf.min_element_size).axis(-2);
+
+  // Min_size
+  std::vector<float> box_min;
+  for (auto i : conf.min_size) {
+    float box_width = i * 0.5;
+    float box_height = i * 0.5;
+    float first = -box_width / conf.IW;
+    float second = -box_height / conf.IH;
+    box_min.push_back(first);
+    box_min.push_back(second);
+    box_min.push_back(-first);
+    box_min.push_back(-second);
+  }
+  TensorShape shape(DType::FLOAT32, {conf.min_element_size, 4});
+  Buffer buffer_min(shape);
+  buffer_min.copy_from(box_min.data());
+  auto Box_min = edsl::Constant(buffer_min, "box_min");
+
+  auto C_min_size = C + Box_min;
+  C_out = C_min_size;
+
+  // Max_size
+  edsl::Tensor C_max_size;
+  int max_element_size = attrs.max_size.size();
+  int result_size = max_element_size < conf.min_element_size ? max_element_size : conf.min_element_size;
+  if (attrs.scale_all_sizes && max_element_size > 0) {
+    std::vector<float> box_max;
+    for (size_t i = 0; i < result_size; i++) {
+      float box_width = std::sqrt(conf.min_size[i] * attrs.max_size[i]) * 0.5f;
+      float box_height = box_width;
+      float first = -box_width / conf.IW;
+      float second = -box_height / conf.IH;
+      box_max.push_back(first);
+      box_max.push_back(second);
+      box_max.push_back(-first);
+      box_max.push_back(-second);
+    }
+    TensorShape shape_max(DType::FLOAT32, {result_size, 4});
+    Buffer buffer_max(shape_max);
+    buffer_max.copy_from(box_max.data());
+    auto B_max = edsl::Constant(buffer_max, "box_max");
+
+    C_max_size = op::repeat(op::unsqueeze(C_mix_box, {-2})).count(result_size).axis(-2) + B_max;
+
+    // In this case, process index which min_size and max_size both have
+    if (conf.min_element_size <= max_element_size) {
+      C_out = op::concatenate({C_min_size, C_max_size}, -1);
+    } else {
+      C_out = op::concatenate(
+          {edsl::reshape(
+               op::concatenate(
+                   {op::slice(C_min_size).add_dim(0, conf.H).add_dim(0, conf.W).add_dim(0, result_size).add_dim(0, 4),
+                    C_max_size},
+                   -1),
+               {conf.H, conf.W, result_size * 8}),
+           edsl::reshape(op::slice(C_min_size)
+                             .add_dim(0, conf.H)
+                             .add_dim(0, conf.W)
+                             .add_dim(result_size, conf.min_element_size)
+                             .add_dim(0, 4),
+                         {conf.H, conf.W, (conf.min_element_size - result_size) * 4})},
+          -1);
+    }
+  }
+
+  // Aspect_ratio
+  if (conf.aspect_ratios_scale_size.size() > 0) {
+    std::vector<float> min_size_ar;
+    if (attrs.scale_all_sizes) {
+      for (auto ms : conf.min_size) min_size_ar.push_back(ms);
```

Comment:
Are you making a copy here?  You can use 
std::vector<float> min_size_ar=conf.min_size;